src/main/java/core/mvc/HandlerInterceptorRegistry.java

```diff
@@ -0,0 +1,28 @@
+package core.mvc;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HandlerInterceptorRegistry {
```

Comment:
`HandlerInterceptorRegistry` 👍 
단일 객체와 복합 객체를 모두 동일하게 다루도록 단일 객체와 같은 인터페이스를 구현해도 좋을 것 같아요 :)
[참고 - composite pattern](https://www.baeldung.com/java-composite-pattern)
```suggestion
public class HandlerInterceptorRegistry implements HandlerInterceptor {
```